docs/developing.md

```diff
@@ -32,6 +32,43 @@ cd $GOPATH/src/openshift
 git clone https://github.com/openshift/hive.git
 ```
 
+## Deploying with Kubernetes on Docker (kind)
+
+[Kind](https://github.com/kubernetes-sigs/kind) can be used as a lightweight development environment for deploying and testing Hive. The following instructions cover creating an insecure local registry (allowing for dramatically faster push/pull), and configuring your host OS, as well as the kind cluster to access it. This approch runs Hive in a container as you would in production, giving you the best coverage for manual testing.
+
+This approach requires [Docker](https://docs.docker.com/install/linux/docker-ce/fedora/). At present we do not have kind working with podman.
```

Comment:
How about use `https://docs.docker.com/install` as the docker link, as someone may also want to run on Ubuntu etc.

Reply:
:+1: